Modularize cloning section (#4060)
theforeman/foreman-documentation: guides/common/modules/proc_cloning-without-pulp-data.adoc

:_mod-docs-content-type: PROCEDURE

[id="cloning-without-pulp-data"]
= Cloning without Pulp data

> You can clone {Project} server without including Pulp data.

Reviewer: ```suggestion
You can clone {ProjectServer} without including Pulp data.
```

.Procedure

> Follow the steps in the procedure in xref:preparing-the-source-server-for-cloning[] and xref:cloning-to-the-target-server[] replace the steps that involve cloning with Pulp data with the following steps:

Reviewer: I suggest to rephrase this; maybe you can use it as prerequisite. 

Alternative way: You can split sets of steps into snippets and include whatever your need in this and the other procedure. Note that this leads to more duplicate rendered docs.

. Perform a backup with PostgreSQL databases active excluding the Pulp data:
+
[options="nowrap" subs="attributes"]
----
# {foreman-maintain} backup offline --skip-pulp-content \
--assumeyes /var/backup
----

> +

Reviewer: ```suggestion
```

. Stop and disable {Project} services:
+
[options="nowrap" subs="attributes"]
----
# {foreman-maintain} service stop
# {foreman-maintain} service disable
----

> +

Reviewer: ```suggestion
```
